frontend/src/pages/middlewares/restrict-login-signup.tsx

```
@@ -10,6 +16,59 @@ const QueryParamsSchema = z.object({
+
+    setHasConsented(true);
+  };
+
+  return (
+    <>
+      {config.authConsentContent && !hasConsented && (
+        <div className="fixed inset-0 z-50 flex items-center justify-center bg-mineshaft-700/80 bg-opacity-90">
+          <div className="max-h-[80vh] w-4/12 overflow-y-auto rounded-lg bg-bunker-800 p-6 text-white">
+            <div
+              // eslint-disable-next-line react/no-danger
+              dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)